docs/requirements/path-unwinding-forwarding-requirements.md

```diff
@@ -0,0 +1,99 @@
+
+For example, ATOM is native to the Cosmos Hub, a user could transfer ATOM to Osmosis and deposit in a liquidity pool to earn yield on this token. After depositing their ATOM into a pool on Osmosis, a better yield opportunity could arise, for example a better pool APY on another Cosmos DEX, e.g. Crescent or a better yield on lending ATOM on Umee. The user would then want to transfer the ATOM from Osmosis to Crescent (or Umee).
+
+### 2. Transferring liquid staking derivatives
+
+Liquid staking derivatives are minted on liquid staking zones and represent a staked asset. There is a common misconception from users that these derivatives originate on the chain of the original staked token. This results in users sending derivatives back to the chain of the natively staked token and then onto the next destination. 
+
+For examples, a user has stATOM on Osmosis, they want to move the stATOM to Evmos, instead of going from Osmosis --> Stride --> Evmos, a user tries to unwind themself and routes the tokens Osmosis --> Cosmos Hub --> Evmos. 
+
+### 3. Moving a token that originated from an interoperability zone or chain, or asset issuer
+
+Tokens that originate from other blockchain ecosystems that don't yet support IBC, flow into the Cosmos ecosystem through interoperability zones. These tokens are then sent onto other chains with a specific use case for these tokens and a user could want to move this token from one chain to another. 
+
+For example, ETH from Ethereum flows into Osmosis via Axelar, where the final step moving ETH from Ethereum to Osmosis uses an ICS-20 transfer from Axelar to Osmosis. A user may then want to move ETH from Osmosis onto another chain, for example Injective. However, the path with 1 hop would be a transfer from Axelar to Injective.
+
+### 4. Moving a token from one chain to another, swapping the token and transferring it onwards to a new destination
+
+A user on one chain, for example the Cosmos Hub holds an asset, e.g. ATOM and wants to instead have AKT on Akash. The user must transfer to ATOM to a DEX chain, swap the ATOM for AKT and then send the AKT onwards to Akash. 
+
+# Functional requirements
+
+## Assumptions and dependencies
+
+1. A functional relayer implementation is required for this feature.
+2. Routing information for the final hop for unwinding or for forwarding is configured for the user through a front end client, or configured by another means using off-chain data. 
+3. The feature will have no impact on the existing function of a typical ICS-20 transfer where a native token is sent to another chain.
+4. Fees will be treated the same as with other IBC applications.
```

Comment:
It's not possible to atomically incentivize the whole path yet. It does seem theoretically possible once the port router refactor is in place, but I haven't given it in depth thought

Reply:
modified to change the scope of requirements to ignore fees